com.viber.Viber.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "com.viber.Viber",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "viber",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=home",
+    /* needed for video chat */
+    "--device=all",
+    "--talk-name=org.freedesktop.Notifications",
+    "--extra-data=viber.deb:bacc38034fa6d926e25122c152c867a4f576b59e920af58ce904c4e9f255773f:84861076::http://download.cdn.viber.com/cdn/desktop/Linux/viber.deb"
+  ],
+  "modules": [
+    {
+      "name": "viber",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 apply_extra /app/bin/apply_extra",
+        "install -Dm755 viber /app/bin/viber",
+        "install -Dm644 com.viber.Viber.appdata.xml /app/share/appdata/com.viber.Viber.appdata.xml",
+        "cp /usr/bin/ar /app/bin",
+        "cp /usr/lib/libbfd-*.so /app/lib"
+      ],
+      "sources": [
+        {
+          "type": "script",
+          "dest-filename": "apply_extra",
+          "commands": [
+              "ar x viber.deb",
+              "rm -f viber.deb",
+              "tar xf data.tar.xz",
+              "rm -f control.tar.gz data.tar.xz debian-binary",
+              "mv opt/* .",
+              "mv usr/* .",
+              "chmod -R a-s,go+rX,go-w .",
+              "rmdir usr",
+              "mkdir -p export/share/applications",
+              "sed 's|Icon=/usr/share/pixmaps/viber.png|Icon=com.viber.Viber|; s|Exec=/usr/bin/viber|Exec=viber|' share/applications/viber.desktop > export/share/applications/com.viber.Viber.desktop",
+              "echo StartupWMClass=Viber >> export/share/applications/com.viber.Viber.desktop",
```

Comment:
`desktop-file-edit --set-key=StartupWMClass --set-value=Viber export/share/applications/com.viber.Viber.desktop`

is probably more robust.

Reply:
thanks